README.md

```diff
@@ -220,6 +253,46 @@ typical setup.
 }
 ```
 
+<a name="performance"></a>
+## Performance
+
+<a name="specificity"></a>
+### Specificity
+
+Although in the name (cascading style sheets) cascading can introduce
```

Comment:
The reason its explicit with (cascading style sheets) its because its making a point for the cascading part. I wouldn't change that.